node/utils/updateOrganizationRequest.ts

```diff
@@ -166,4 +173,6 @@ export const updateOrganizationRequest = async (
       }
     }
   }
+
+  return null
```

Comment:
Why are we adding this?

Reply:
Otherwise we get this: 
`Not all code paths return a value.ts(7030)`

Reply:
If you delete the `return { status: 'success', message: '', id: organizationId }` line above then you should be able to get rid of this as well.

Reply:
✅ It fixed it, thank you!